pkg/controller/garbagecollector/garbagecollector.go

```diff
@@ -43,6 +43,8 @@ import (
 
 const ResourceResyncTime = 30 * time.Second
 
+const OrphanFinalizerID = "orphanFinalizer"
```

Comment:
Maybe move to another package... seems bad to make people import the guts of the garbage collector to use this constant.